packages/deployment/src/shared/Artifacts.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs')
```

Comment:
Just wondering, can't we use the bre to get the truffle artifact directly? Or is this not possible because we're in a different project namespace and we're not requiring buidler to compile anything?